templates/shakes/_sidebar.html

```diff
@@ -156,7 +156,7 @@ <h3>Followed by <span class="count">({{follower_count}})</span></h3>
         {% for follower in followers %}
           <li>
             <a href="/user/{{follower.name}}" title="{{follower.display_name()}}">
-              <img class="avatar--img" src="{{follower.profile_image_url()}}" width="48" height="48">
+              <img class="avatar--img" src="{{follower.profile_image_url()}}" width="48" height="48" alt="{{follower.name}}">
```

Comment:
How is this handled by screen readers? Here's a case where the img tag has a link with a title attribute. Which is used for vocalization? The title or the alt text? Both?

Reply:
Good question! I just tested, and I'm happy to say that the screen reader handles it well — it prioritizes `alt` over the `title`, and it doesn't say it twice if you have both.

That said, you raise a good question. I didn't remove the `title` attributes because some people like the tooltip it provides for mouse users, but certainly all the a11y reading I'm doing says not to use them: https://developer.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/

My take: Adding the `alt` is mandatory, and having both `alt` and `title` doesn't cause any problems for screen readers. I'd be okay with leaving them both, but if we're uncomfortable with it, we should remove the `title` attributes from these three instances, at the cost of no longer having tooltips for mouse users.